tests/integration/telemetry/tracing/bootstrap/main_test.go

```diff
@@ -0,0 +1,123 @@
+//go:build integ
+// +build integ
+
+// Copyright Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bootstrap
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/pkg/test/util/retry"
+	"istio.io/istio/tests/integration/telemetry/tracing"
+)
+
+func TestMain(m *testing.M) {
+	framework.NewSuite(m).
+		Label(label.CustomSetup).
+		Setup(istio.Setup(tracing.GetIstioInstance(), setupConfig)).
+		Setup(tracing.TestSetup).
+		Run()
+}
+
+func setupConfig(_ resource.Context, cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+	cfg.ControlPlaneValues = `
+meshConfig:
+  defaultConfig:
+    proxyMetadata:
+      DISABLE_BOOTSTRAP_TRACING: "true"
```

Comment:
Why is line 52 needed? Line 56 has configured DISABLE_BOOTSTRAP_TRACING="true".

Reply:
for `pilot-agent` not generate static cluster like `zipkin`

Reply:
getProxyConfigOptions->toTemplateParams->WriteTo->CreateFile->initializeEnvoyAgent